shared/javascripts/local_storage.js

```diff
@@ -87,14 +87,20 @@ var ls = {
   }
 };
 
-// Determine whether localstorage can be used directly
-try { 
-  localStorage;
+try {
+  // check for jetpack runtime environment
+  exports.ls = ls;
```

Comment:
I should change it to `module.exports`(CommonJS) and I think the comment is a bit misleading(I really need to just import it as a commonjs module, but that doesn't really check for Jetpack runtime environment).